src/Components/ClusterRules/ClusterRules.spec.ct.js

```diff
@@ -3,9 +3,54 @@ import { mount } from '@cypress/react';
 import { IntlProvider } from 'react-intl';
 import { MemoryRouter, Route } from 'react-router-dom';
 import { Provider } from 'react-redux';
+import filter from 'lodash/filter';
+import map from 'lodash/map';
+import uniq from 'lodash/uniq';
+import sortBy from 'lodash/sortBy';
+import reverse from 'lodash/reverse';
+import flatMap from 'lodash/flatMap';
```

Comment:
hm, here, it seems it worked to you right? :D did you have any complaints from cypress while running this?

Reply:
No complaints, it is working fine. Thanks for the advice